templates/authorizer.py

```diff
@@ -61,33 +63,40 @@
 ))
 template.set_parameter_label(param_use_cert, "Use TLS certificate")
 
-param_auto_use_ef_idp = template.add_parameter(Parameter(
```

Comment:
Are you sure you want to roll back theses changes?

Reply:
Yes... 
I want this version to deploy the existing stacks without changes.
The next iteration will be for switching over to the UserPool with Azure AD.